tests/output/vt100_support.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
The file should be .inc, not .php. Otherwise there might be issues when temporary files are cleaned up.

Thanks.